# encoding: utf-8
# module scipy.signal._max_len_seq_inner
# from /.venv/lib/python3.8/site-packages/scipy/signal/_max_len_seq_inner.cpython-38-aarch64-linux-gnu.so
# by generator 1.147
# no doc
# no imports

# functions

def _max_len_seq_inner(int64, *args, **kwargs): # real signature unknown; NOTE: unreliably restored from __doc__ 
    """
    Supported prototypes:
    
        - _max_len_seq_inner(int64[:], int8[:], int, int, int8[:])
        - _max_len_seq_inner(int32[:], int8[:], int, int, int8[:])
    """
    pass

# no classes
# variables with complex values

__loader__ = None # (!) real value is '<_frozen_importlib_external.ExtensionFileLoader object at 0xffff92cb1d90>'

__pythran__ = (
    '0.12.1',
    '2023-02-19 18:20:36.519580',
    '8ba55676ef5470ea0efe586fbe688e9b708e4c3cc1ae7b410298a8869f8cdbf8',
)

__spec__ = None # (!) real value is "ModuleSpec(name='scipy.signal._max_len_seq_inner', loader=<_frozen_importlib_external.ExtensionFileLoader object at 0xffff92cb1d90>, origin='/.venv/lib/python3.8/site-packages/scipy/signal/_max_len_seq_inner.cpython-38-aarch64-linux-gnu.so')"

